libraries/lib-time-and-pitch/StaffPadTimeAndPitch.cpp

```diff
@@ -50,11 +50,9 @@ StaffPadTimeAndPitch::StaffPadTimeAndPitch(
 void StaffPadTimeAndPitch::GetSamples(float* const* output, size_t outputLen)
```

Comment:
My first comment on this PR.

I did not review the PR that added this library, so I must become familiar with that first.

I see `libraries/lib-time-and-pitch/CMakeLists.txt` lacks a long-form CMake comment at the top, which is the convention with other libraries.

Please supply that comment, about the general purpose of the library.

Reply:
Done.